### PR TITLE
fix LND gpg keys download

### LINKED ISSUE #660: Wrong lines in installation guide (EDIT: probably wget -O instead of wget -o)
In the installation guide [here](https://stadicus.github.io/RaspiBolt/raspibolt_40_lnd.html) you say

```
$ pgp ./roasbeef.asc
> 9769140D255C759B1EB77B46A96387A57CAAE94D
$ gpg ./bitconner.asc
> 9C8D61868A7C492003B2744EE7D737B67FA592C7
```

I get the error: 

```
admin@RPi4-2:/tmp $ pgp ./bitconner.asc
gpg: WARNING: no command supplied.  Trying to guess what you mean ...
gpg: no valid OpenPGP data found.
gpg: processing message failed: Unknown system error
WARNING: Can't find the right public key-- can't check signature integrity.

```

If I cat the file I get: 

```
admin@RPi4-2:/tmp $ cat roasbeef.asc
--2020-11-24 10:01:55--  https://keybase.io/roasbeef/pgp_keys.asc
Resolving keybase.io (keybase.io)... 34.238.229.11, 34.238.91.171
Connecting to keybase.io (keybase.io)|34.238.229.11|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 6900 (6.7K) [text/plain]
Saving to: ‘pgp_keys.asc’

     0K ......                                                100%  129M=0s

2020-11-24 10:01:56 (129 MB/s) - ‘pgp_keys.asc’ saved [6900/6900]

```
So this file seems to be just the output-log. I am not familiar with pgp. 

In the video below the text it does not show this part. It continues with: 

```
gpg ./pgp_keys.asc
gpg --import ./pgp_keys.asc 
```

So I suppose the text is incorrect. But since this seems to be an encryption thing i would like to sort it out.

EDIT: 

Is it possible that you wante to use `wget -O FILE` and instead wrote the instructions with `wget -o FILE` so that instead of saving the downloaded key as this file the log gets saved?

### PR DESCRIPTION
**Pull request by Stadicus**
Commit 8e3810afb6c31c1f27dcbb0c69983d6b9101f10a changed how the PGP keys of LND maintainers are downloaded. The commands are not correct and use a lowercase `-o` instead of param `-O`.

This commit

* changes `wget -o` to `wget -O` to get the plaintext PGP keys
* fixes #660

### CODE REVIEW

**Comment by bvrulez** [comment]
Looks good. 

Also, the manual references `9769140D255C759B1EB77B46A96387A57CAAE94D` for the output of `pgp ./roasbeef.asc` but I get `E4D85299674B2D31FAA1892E372CBD7633C61696` as stated by me [here](https://github.com/Stadicus/RaspiBolt/issues/660#issuecomment-732823230)

I can not easily find something on the command `pgp KEYFILE` but I think it is a hash for the key to make the user check if he got the right one. I found a discussion where roasbeef updated his key so it's probably just an old hash that is referenced. 

The third issue (in this otherwise flawless guide) is this part:

```
gpg --verify roasbeef-manifest-v0.11.1-beta.txt.sig manifest-v0.11.1-beta.txt
> gpg: Good signature from "Olaoluwa Osuntokun <laolu32@gmail.com>" [expired]
> gpg: Note: This key has expired!
> Primary key fingerprint: 9769 140D 255C 759B 1EB7  7B46 A963 87A5 7CAA E94D
>      Subkey fingerprint: 4AB7 F8DA 6FAE BB3B 70B1  F903 BC13 F65E 2DC8 4465
```

I did get the subkey fingerprint, but not the primary key. See [here](url).

**I was able to set everything up despite this error.**

**Comment by Stadicus** [comment]
Thanks for testing, I now opted to remove roasbeef's PGP key, as LND now uses Connor's signature officially. No need to verify
additional signatures that are not officially supported signing keys.

PTAL

**Comment by bvrulez** [comment]
Just found this which is older and relates to the same lines in the code: https://github.com/Stadicus/RaspiBolt/pull/657#issue-520972856

File: bitconner.asc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (lucasmoten) [inline bitconner.asc:1]: Not necessary to include this file as it's being downloaded. When following full install it should have ended up in /tmp

**Review by vindard — APPROVED** [review]
I ran into the `-o` issue and roasbeef's outdated keys issue as well. This looks about right for what I did on my end to address :+1: 

LGTM :tada:

**Comment by Stadicus** [comment]
Fixed now, sorry that it took so long!

**Pull request closed without merging**